lib/common/rand.c

```diff
@@ -0,0 +1,55 @@
+#include <openssl/rand.h>
```

Comment:
Please add the MIT copyright boilerplate, with year set to 2021 and the copyright owners set to the people who have written the code (or who own the right).